src/Tasks/SynTree/Config.hs

```diff
@@ -79,9 +80,18 @@ checkSynTreeConfig SynTreeConfig {..}
     | minNodes < atLeastOccurring * 2 - 1 = reject $ do
         english "Your minimum number of nodes does not permit enough leaves for all desired literals."
         german "Minimale Anzahl der Blätter ist zu niedrig um alle Literale zu verwenden."
+    | minDepth < 1 = reject $ do
+        english "Minimal depth must be positive"
+        german "Minimale Tiefe muss positiv sein."
+    | maxNodes < minDepth = reject $ do
+        english "Your maximum number of nodes is less than what your minimum depth requires."
+        german "Maximale Anzahl der Blätter würde eingestelle minimale Tiefe verletzen."
+    | maxDepth < minDepth = reject $ do
+        english "Maximal depth must not be smaller than minimal depth"
+        german "Maximale Tiefe ist kleiner als minimale Tiefe"
     | maxNodes > maxNodesForDepth maxDepth = reject $ do
         english "Your minimum number of nodes is larger than what your maximum depth enables."
-        german "Minimale Anzahl der Blätter würde eingestellte Tiefe verletzen."
+        german "Minimale Anzahl der Blätter würde eingestellte maximale Tiefe verletzen."
```

Comment:
Like in the `maxConsecutiveNegations < 0` case you fixed above, also here the current reference to "minimum" is wrong?